src/test/java/org/retest/rebazer/RepositoryHostingTypesTest.java

```diff
@@ -0,0 +1,50 @@
+package org.retest.rebazer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.junit.Test;
+import org.retest.rebazer.connector.BitbucketConnector;
+import org.retest.rebazer.connector.GithubConnector;
+import org.retest.rebazer.connector.RepositoryConnector;
+import org.retest.rebazer.domain.RepositoryConfig;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+
+public class RepositoryHostingTypesTest {
+	@Test
+	public void should_return_correct_bitbucket_defaultUrl() throws MalformedURLException {
```

Comment:
1. Always throw `Exception`.
1. Test method names should be like "Given When Then". That is, in front of "should", there should be some sort of precondition. For instance: `bitbucket_type_should_result_in_default_bitbucket_url`.